ocs_ci/ocs/cluster.py

```diff
@@ -1022,6 +1022,14 @@ def delete_blockpool(self, pool_name):
         time.sleep(30)
         self.RBD.exec_oc_cmd(f"patch {patch}")
 
+    def ceph_config_set_debug(self):
```

Comment:
modify to accept and set the debug level instead of hard coding

Reply:
made changes accordingly.